core/quill.js

```diff
@@ -207,6 +213,7 @@ class Quill {
   }
 
   setContents(delta, source = Emitter.sources.API) {
+    this.toggleEmpty(this.getLength() <= 1);
```

Comment:
Yeah, I moved this to the constructor since the method needs to be called on init as well